pkg/block/indexheader/header_test.go

```diff
@@ -519,78 +521,3 @@ func readSymbols(bs index.ByteSlice, version, off int) ([]string, map[uint32]str
 	}
 	return symbolSlice, symbols, errors.Wrap(d.Err(), "read symbols")
 }
-
-func TestIndexHeaderV1LookupSymbols(t *testing.T) {
```

Comment:
Is this for the old TSDB 1 format?

Reply:
I think yes, we added it recently but having Symbols as an interface just for this one test isnt much bang for the buck i figured.